src/UglyToad.PdfPig/Annotations/AnnotationProvider.cs

```diff
@@ -37,11 +40,12 @@ public IEnumerable<Annotation> GetAnnotations()
                 var type = annotationDictionary.Get<NameToken>(NameToken.Subtype, tokenScanner);
 
                 var annotationType = type.ToAnnotationType();
-                var rectangle = annotationDictionary.Get<ArrayToken>(NameToken.Rect, tokenScanner).ToRectangle(tokenScanner);
+                var rectangle = matrix.Transform(annotationDictionary.Get<ArrayToken>(NameToken.Rect, tokenScanner).ToRectangle(tokenScanner));
 
                 var contents = GetNamedString(NameToken.Contents, annotationDictionary);
                 var name = GetNamedString(NameToken.Nm, annotationDictionary);
-                var modifiedDate = GetNamedString(NameToken.M, annotationDictionary);
```

Comment:
I will have to revert this change, reading up on the modified date in the PDF reference 8.4.1 which indicates this field can basically contain anything.